### ISSUE TITLE
MacBookPro10,1 clarification about latest OCLP version and FileFault

### ISSUE THREAD

**Issue by Ribero** [dortania/OpenCore-Legacy-Patcher#341]
Hi the team,

Can you please clarify the situation about systems like the MacBookPro10,1 (Intel HD4000 GPU) that seems to be working with all security features activated (SIP and SecureBootModel) under BigSur and specifically the fact that FileVault is enabled (which is probably one of the most important feature of OCLP over other patching approach).

So latest version of OCLP (0.19+ ?) that begins the support Monterey seems to disable SIP and SBM and as a result prevent FileFault from working for systems running Big Sur.

Is that true ?

If true, could you provide a way to support BigSur with all security feature activated and no root patching and provide Monterey support with all the needed root patching with a specific setting in OCLP ?

In other words, how to benefits of the latest version of OCLP with all security features activated (no root patching and FileVault) for systems that only wants to run BigSur and do not want to downgrade security ?

Thanks a lot in advance for the clarification.

**Comment by Ausdauersportler** [comment]
It is better to Discord to ask such question (it is not really an issue) and you are free to set SMB to true manually with all older, recent, and upcoming OCLP versions all the time. And this question came up several times, now. One can search the closed issues :-)

The change has been made to make it possible to add GPU patches for Monterey for the HD4000. The logic is currently: If you detect a system which could need root patching disable SBM.

**Comment by Ribero** [comment]
Sorry, discord on safari on iOS is nearly unusable but I will try again.

I saw that the question has been raised before wut I did not found a clear answer and unfortunately your answer is not really more clear.

Sorry to ask again but : How to benefits of the latest version of OCLP with all security features activated (no root patching and FileVault) for systems that only wants to run BigSur and do not want to downgrade security ?

**Comment by Ausdauersportler** [comment]
You can either enable SBM using the OCLP app **or** you can read the OpenCore online documentation how to achieve this editing the config.plist manually. 

`5.  Patcher Settings`

then
`6.  Set SIP and SecureBootModel:	SIP: True SBM: True`

then

```
Valid options:

1. Enable Both
2. Disable SIP only
3. Disable SecureBootModel Only
4. Disable Both

        
Set SIP and SecureBootModel(ie. 1): 

```
You are really discussing an artificial non existing problem here!

**Comment by Ribero** [comment]
Thanks for sharing the settings.

So if one update to the latest OCLP a system running FileVault on BS it will not face any issue ? I understood that OCLP new default settings were going to break existing well running systems using FileVault and so it was worth to open an issue due to potential problematic situation it could lead to.

But I may be wrong so this issue can be closed. 

Many thanks for your help.

**Comment by Ribero** [comment]
Some updates: 

In advanced settings:

```
Valid options:

1. Enable Both
2. Disable SIP only
3. Disable SecureBootModel Only
4. Disable Both
```

Selecting 1 (Enable both) does not work as expected. SIP is still disabled.
I faced this because my laptop in enrolled in Microsoft InTunes to be part of company managed devices (which requires SIP and FileFault to be enabled).

After using the latest version of OCLP with manually enabling SIP/SBM my device was no more assumed as conformed  and was on the way to be deactivated.

I reverted to OCLP v0.1.6 and re-enrolled the device which became conform again.

So I think this problem can now be considered as a real issue.

**Comment by khronokernel** [comment]
As discussed in a previous issue, multi boot configuration breaks heavily if we try to support per-OS configs. Patcher Settings are still valid, reason your machine reported SIP disabled is that the old SIP value remained. Resetting NVRAM resolves this

Closing as documentation reflects this

As a final note, FileVault does not require SIP or SecureBootModel to be enabled in Big Sur.

**Comment by Ribero** [comment]
My machine use only one system (Big Sur) and only one boot disk (a FileVault one). 
I do not boot on another system even on external drive, i do not test Monterey, i do not use windows. 

I just upgraded my running system from OCLP v0.1.6 to latest and it broke the security.

**Comment by Ribero** [comment]
Finally got time to test again and installed v0.2.2, manually enabled SIP and SBM, reboot, reset NVRAM
-> Microsoft InTunes does not report anymore that the laptop is not compliant.

<img width="912" alt="image" src="https://user-images.githubusercontent.com/2266264/124243235-ec4dfa80-db1d-11eb-8700-fbe7bcdfa974.png">

By the way let me thanks again all the team for the fantastic work made that allow us to used beloved devices on Big Sur. I personally use it as my main computer with absolutely no issue, as if it was officially and fully supported by Apple. Even SideCar over WiFi is working well, incredible ! Thanks.
